collectors/0/pxc-collector.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+"""
```

Comment:
Sure, changed the header already and confirmed the Agreement. I'll need some time for making the other changes, but I'll take care of your suggestions.